### ISSUE TITLE
S3 deployment calcSourceHash fails on folder not found

### ISSUE THREAD

**Issue by sam-goodwin** [aws/aws-cdk#4445]
S3 deployment is crashing with no such file found.

### Reproduction Steps

```bash
cdk init --language=typescript
```
```typescript
import cdk = require('@aws-cdk/core');
import s3 = require('@aws-cdk/aws-s3');
import s3Deploy = require('@aws-cdk/aws-s3-deployment');

import path = require('path');

export class TestStack extends cdk.Stack {
  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
    super(scope, id, props);

    const bucket = new s3.Bucket(this, 'Bucket');
    new s3Deploy.BucketDeployment(this, 'Deploy', {
      destinationBucket: bucket,
      sources: [s3Deploy.Source.asset(path.join(__dirname, 'web'))]
    });
    // The code that defines your stack goes here
  }
}
```

The src folder is not included in the distributed `tgz` of `@aws-cdk/aws-s3-deployment`.

### Error Log
```
NOENT: no such file or directory, scandir '/Users/samgoodwin/workspaces/test/node_modules/@aws-cdk/aws-s3-deployment/lambda/src'
````

### Environment

  - **CLI Version      :** 1.12.0
  - **Framework Version:** 1.12.0
  - **OS               :** Mac OS
  - **Language         :** TypeScript
--- 

This is :bug: Bug Report

**Comment by sam-goodwin** [comment]
Looks to have been caused by https://github.com/aws/aws-cdk/commit/5225306c4a83d78593c1c0a04367c845b6817be4

**Comment by sam-goodwin** [comment]
This error would not be found when locally developing as it only shows up when npm installing the package. Perhaps we need to update the `.npmignore` file to include the src folder in the tar?

**Comment by eladb** [comment]
Apologies about this. Will be fixed ASAP. Duplicate of #4404 fixed by #4430

**Comment by sam-goodwin** [comment]
Didn't see that issue. Thanks for the quick fix!
